gabbi/tests/test_unsafe_yaml.py

```diff
@@ -33,6 +33,9 @@
 
 
 yaml.add_constructor(u'!IsNAN', lambda loader, node: util.NanChecker())
+yaml.add_constructor(
```

Comment:
I'll fix this on master, as it is breaking there too. It's because of the new release of yaml. I'll need to update the requirements to make everything smooth.

Reply:
Turns out this is considerably more complicated than I thought. #252 is in progress to try to fix it.

Reply:
Oh that makes sense why I couldn't reproduce the error. I'll wait for #252 before making a change to remove my last commit one (56c9471).

Reply:
I've changed master to not use PyYAML above >=4.0 for the time being, 4.1 is probably going to change.

If you remove your last patch, rebase onto master (or merge from it), I think we'll be good to go
and we can release this change and the yaml guard. Once I figure out what's really going on with 4.x we can look back at #252